example/app/data/DemoDataSource/DemoPackage/blueprints/Plugins/View/RecipeViewConfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "dmss://system/SIMOS/Blueprint",
+  "name": "RecipeViewConfig",
```

Comment:
Is "InlineRecipeViewConfig" or "ProvidedRecipeViewConfig" a better word? :thinking:

Reply:
I liked this, it is much more precise. 

I renamed it to InlineRecipeViewConfig, but I liked the other alternative also.